packages/http/src/validator/__tests__/functional.spec.ts

```diff
@@ -41,13 +48,22 @@ describe('HttpValidator', () => {
           await validator.validateInput({
             resource: httpOperations[2],
             input: BAD_INPUT,
-            config: {
-              mock: false,
-              validate: { request: { headers: false, query: true, body: false } },
-            },
+            config: { mock: false, validate: { request: { headers: false, query: true, body: false } } },
           }),
         ).toMatchSnapshot();
       });
+
+      describe('when all required params are provided', () => {
+        it('returns no validation errors', async () => {
+          expect(
+            await validator.validateInput({
+              resource: httpOperations[0],
+              input: GOOD_INPUT,
+              config: { mock: false, validate: { request: { headers: false, query: true, body: false } } },
+            }),
+          ).toMatchSnapshot();
```

Comment:
Please use `.toEqual([])` instead of `.toMatchSnapshot()` for simple expectations. E.g. this time the expectation is an empty array, so it would be way more readable when the expected result was in test code directly.